main/test/flix/Test.Exp.Foreach.flix

```diff
@@ -0,0 +1,7 @@
+namespace Test/Exp/Foreach {
+
+    @test
+    def testForeach01(): Int32 = // Dummy program that just checks compiler phases
+        foreach (x <- xs) x + 1
```

Comment:
Now you can write some test cases to experiment with what you think should be allowed :)